source/includes/steps-install-shell-macos.yaml

```diff
@@ -0,0 +1,55 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-macos
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: open-mdb-shell-page
+ref: open-mdb-shell-page-macos
+---
+title: "Download the tarball."
+ref: download-tarball
+level: 4
+content: |
+---
+title: "Extract the files from the downloaded archive."
+ref: extract-tarball
+level: 4
+content: |
+   
+   .. code-block::
+
+      tar -zxvf <path to your mongodb shell .tgz>
+
+   If your web browser automatically unzips the file as part of the
+   download, the file would end in ``.tar`` instead.
+---
+title: "Add the MongoDB Shell binary to your ``PATH`` environment
+   variable."
+ref: add-shell-to-path
+level: 4
```

Comment:
I think we should leave as-is. I think this will prevent us down the line from making mention of "hey to run this command, make sure mongosh is in your path"